interface/patient_file/summary/add_edit_issue.php

```diff
@@ -806,8 +808,18 @@ function divclick(cb, divid) {
                     </div>
                 </div>
                 <!-- End of reaction -->
+                <!-- Verification Status for Medication Allergy -->
+                <div class="form-group" id='row_verification'>
+                    <label class="col-form-label col-4" for="form_verification"><?php echo xlt('Verification Status'); ?>:</label>
+                    <div class="col-10">
```

Comment:
This is actually incorrect. The Bootstrap 4 grid system goes all the way up to col-12, not col-14.